diag-main.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/bash
+
+PACKAGE_TREE="$(dirname $(dirname $(readlink -f $0)))"
+run() {
+  if [[ $1 == "nfs" ]] || [[ $1 == "cifs" ]];
+  then 
+    if [[ $1 == "nfs" ]]
+    then script_path="${PACKAGE_TREE}/lib/NfsDiagnostics/nfsclientlogs.sh"
```

Comment:
Nitpick: I'm a little divided about pushing nfsclientlogs (and the smb equivalent) into `lib`. I figure that since they're executable, they should be under `bin`. Then again, I also see that having them in `lib` makes sense. I'm not going to make this a blocker, but I guess it is something we could ask for an opinion about with other folks.